source/billing.txt

```diff
@@ -26,14 +26,15 @@ Overview
 --------
```

Comment:
While we're in this file, can we remove the "Overview" structure? We're moving away from this style. Totally optional if you don't want to complicate the PR more than needed.

If you opt to change it, you'll need to change the two subheading levels.